src/main/java/com/mgmtp/radio/service/station/StationOnlineServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+package com.mgmtp.radio.service.station;
+
+import com.mgmtp.radio.dto.station.StationDTO;
+import com.mgmtp.radio.dto.user.UserDTO;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Service("stationOnlineService")
```

Comment:
just need @Service for short

Reply:
Lam said, we write full like @Service("stationOnlineService") for clear